api_docs/changelog.md

```diff
@@ -236,16 +239,15 @@ format used by the Zulip server that they are interacting with.
 
 **Feature level 159**
 
-* [`POST /register`](/api/register-queue), [`GET /events`](/api/get-events),
-  `PATCH /realm`: Nobody added as an option for the realm setting
-  `edit_topic_policy`.
-* [`POST /register`](/api/register-queue), [`GET /events`](/api/get-events),
-  `PATCH /realm`: Nobody added as an option for the realm setting
-  `move_messages_between_streams_policy`.
-* [`PATCH /messages/{message_id}`](/api/update-message): Permission to edit stream
-  and topic of messages do not depend on `allow_message_editing` setting now.
-* [`PATCH /messages/{message_id}`](/api/update-message): Message senders are not
-  allowed to edit topics indefinitely now.
+* `PATCH /realm`, [`POST /register`](/api/register-queue),
+  [`GET /events`](/api/get-events):
+  Nobody added as an option for the realm settings `edit_topic_policy`
+  and `move_messages_between_streams_policy`.
+* [`PATCH /messages/{message_id}`](/api/update-message): Permission
+  to edit the stream and/or topic of messages no longer depends on the
+  realm setting `allow_message_editing`.
```

Comment:
(By contrast, this way around makes perfect sense here in the API changelog. Focusing on the new state rather than the old is the normal thing to do in a changelog; so I guess you could say that I think of a "Changes" section in the rest of the API docs as more like a "reverse changelog", where the reader starts from understanding the new state and then wants to precisely understand the old states.)

Reply:
That's helpful context for me when reviewing and revising these. Thanks!

Reply:
Yeah this might be the type of idea that we should take the opportunity to write in our API docs documentation, which I think generally needs a pass for the fact that the majority of work on our API docs is documenting new changes, not new endpoints, these days.